### ISSUE TITLE
Page not found on mvp.sitecore.com

### ISSUE THREAD

**Issue by erwinsmit** [Sitecore/XM-Cloud-Introduction#179]
Open mvp.sitecore.com on Chrome on iOS (v16).

I end up on a "page not found" page:
![image](https://user-images.githubusercontent.com/5425589/198587199-d322b279-c83a-496d-8b4e-22307347b1c5.png)

It works fine on my laptop (chrome). But when I open the page incognito I also get a "page not found" page. See the screenshot below, normal chrome left, incognito window on the right. Very strange behaviour.. 

![image](https://user-images.githubusercontent.com/5425589/198588551-4b477931-a02e-46be-82f1-c6353062ccba.png)

So it almost seems like a middle-ware a/b test issue?

**Comment by sc-ivanlieckens** [comment]
This issue was reported in the MVP Slack channel as well and is related to Accept-Language headers and how the ASP.NET Rendering Host doesn't process certain formats of this header causing a nullreference exception which results in a 404. We've raised this internally under issue nr 555373 and it'll be fixed in the v22 release which we'll be applying as soon as possible.

In the meantime mobile devices may face this issue. Desktop devices have not shown this behaviour.

**Comment by robearlam** [comment]
v22 is scheduled to be released alongside XM 10.4 later in the year. This is blocked until that release happens.

**Comment by sc-ivanlieckens** [comment]
While the Workaround eliminates the issue for now I'll leave this open till we can do the V22 replacement

**Comment by robearlam** [comment]
@erwinsmit  - We recently upgraded to a new version of the SDK, which I hope should have fixed this.

Are you still seeing the issue?

**Comment by erwinsmit** [comment]
Wow, that's a blast from the past! I just tested it on iOS and it's all working fine now!

**Comment by robearlam** [comment]
Yeah, I was just doing some house cleaning earlier and figured it would be fixed, just wanted to double check before closing it.
